CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/BaseTest/Screens/MainScreen.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  MainScreen.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+import XCTest
+
+public class MainScreen{
+    let app: XCUIApplication
+    let photoPaymentButton: XCUIElement
+    let cameraIconButton: XCUIElement
+    let configurationButton: XCUIElement
```

Comment:
[clarification]: Maybe, all of the properties here we can make them lazy vars.
Let's discuss this.